cypress/features/regression/experimental/textboxMultiple.feature

```diff
@@ -12,27 +12,27 @@ Feature: Experimental Textbox multiple component
 
   @positive
   Scenario: Check disabled checkbox for a Textbox multiple component
-    When I open multiple "Experimental-Textbox" component in noIFrame with "textbox" json from "experimental" using "disabled" object name
+    When I open multiple "Experimental Textbox Test" component in noIFrame with "textbox" json from "experimental" using "disabled" object name
```

Comment:
All tests as `disabled`, `readonly` etc which are covered by chromatic should be delete